src/makeCalculator.js

```diff
@@ -36,8 +36,47 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(number) {
+      calculator.result += number;
+
+      return calculator;
+    },
+
+    subtract: function(number) {
+      calculator.result -= number;
+
+      return calculator;
+    },
+
+    multiply: function(number) {
+      calculator.result *= number;
+
+      return calculator;
+    },
+
+    divide: function(number) {
+      calculator.result /= number;
```

Comment:
need to check if number === 0

Reply:
divide: function(number) {
      if (number !== 0) {
        calculator.result /= number;
      }
  
      return calculator;
    },